src/sentry/static/sentry/app/components/group/externalIssuesList.tsx

```diff
@@ -44,13 +44,6 @@ type State = AsyncComponent['state'] & {
 class ExternalIssueList extends AsyncComponent<Props, State> {
   unsubscribables: any[] = [];
 
-  static propTypes = {
```

Comment:
Best practice is to remove proptypes if no jsx components are using this component.